src/dotnet/ToolPackageObtainer/ToolPackageObtainer.cs

```diff
@@ -75,7 +84,8 @@ IProjectRestorer projectRestorer
                             individualToolVersion.WithSubDirectories(packageId).Value).Single()).Name;
                 DirectoryPath concreteVersionIndividualToolVersion =
                     individualToolVersion.GetParentPath().WithSubDirectories(concreteVersion);
-                Directory.Move(individualToolVersion.Value, concreteVersionIndividualToolVersion.Value);
+
+                MoveToVersionedDirectory(concreteVersionIndividualToolVersion, individualToolVersion);
```

Comment:
Should you update the variable names here as well?

Reply:
Please, update the variable names. You fixed the parameters, but these variables still have the old names.

Reply:
Sorry, I missed variable names. This is fixed now